src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="" data-qa="big">
+      <div class="search-container">
+        <img
+          src="images/Search.svg"
+          alt="search-icon-big"
```

Comment:
Це буде читати скрінрідер, тому не розділяй слова дефісами
```suggestion
          alt="search icon big"
```